scene/main/node.cpp

```diff
@@ -2040,6 +2093,20 @@ void Node::_set_owner_nocheck(Node *p_owner) {
 	owner_changed_notify();
 }
 
+void Node::_release_exposed_in_owner() {
+	data.owner->data.owned_exposed_nodes.erase(this);
+}
+
+void Node::_acquire_exposed_in_owner() {
+	ERR_FAIL_NULL(data.owner); // Safety check.
+	if (data.owner->data.owned_exposed_nodes.has(this)) {
+		WARN_PRINT(vformat("Setting node name '%s' to be exposed within scene for '%s', but it's already claimed",
```

Comment:
This looks sus. The warning is copied, but cut in the middle. Also the condition suggests that something different happened here, unrelated to name.